src/main/java/org/springframework/data/domain/PageRequest.java

```diff
@@ -36,13 +37,13 @@ public class PageRequest extends AbstractPageRequest {
 	/**
 	 * Creates a new {@link PageRequest} with sort parameters applied.
 	 *
-	 * @param page zero-based page index, must not be negative.
-	 * @param size the size of the page to be returned, must be greater than 0.
+	 * @param pageNumber zero-based pageNumber index, must not be negative.
```

Comment:
Whoops, seems too much was changed by the refactoring. I'll align these upon merge.